arm-compute/2016-03-30/swagger/compute.json

```diff
@@ -243,7 +243,7 @@
           "VirtualMachineExtensionImages"
         ],
         "operationId": "VirtualMachineExtensionImages_Get",
-        "description": "Gets a virtual machine extension image.",
+        "description": "a virtual machine extension image.",
         "parameters": [
```

Comment:
Sorry if I wasn't clear, currently Autorest only appends Gets or Gets and Sets to properties not operation descriptions